posthog/settings/web.py

```diff
@@ -68,6 +68,7 @@
     "posthog.middleware.per_request_logging_context_middleware",
     "django_structlog.middlewares.RequestMiddleware",
     "django_structlog.middlewares.CeleryMiddleware",
+    "posthog.middleware.Fix204Middleware",
```

Comment:
Was looking through `web.py` and noticed that this middlware is added twice - both here and at the end of the middleware list (added in #32222, which was merged just before this PR). Does this middleware need to be specified twice or was that accidental?

Reply:
Ah, probably an accident, I didn't intend to add it twice